concordium-sdk/src/main/java/com/concordium/sdk/crypto/pointchevalsanders/PSPublicKey.java

```diff
@@ -0,0 +1,29 @@
+package com.concordium.sdk.crypto.pointchevalsanders;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.codec.DecoderException;
+import org.apache.commons.codec.binary.Hex;
+
+/**
+ * Public Key for Pointcheval-Sanders Signing scheme.
+ */
+@RequiredArgsConstructor
+@Getter
+public class PSPublicKey {
```

Comment:
I think it's OK adding this but maybe it should substitute the current `String ipVerifyKey` in the existing class  `IdentityProviderInfo`

Reply:
done